config/locales/en.yml

```
@@ -4,6 +4,7 @@ en:
   no_results_with_filter_and_search: There are no results for '%{for}' and the selected filter.
   no_results_with_filter: There are no results for the selected filter.
   you_cannot_perform_this_action: You cannot perform this action
+  you_dont_have_access_to_this_service: You don't have access to this service
```

Personal preference: (feel free to ignore)
Could this be changed to:
```suggestion
  you_do_not_have_access_to_this_service: You do not have access to this service
```
I'm not a fan of contraction in things like error messages.

---

This has been resolved here https://github.com/DFE-Digital/itt-mentor-services/pull/184/commits/aa850c5ca2aa452233102b5b30dcd7ac478329fd